database/seeds/UsersTableSeeder.php

```diff
@@ -11,16 +11,14 @@ class UsersTableSeeder extends Seeder
      */
     public function run()
     {
-        $admin = BackpackUser::create([
+        $admin = User::create([
             'name' => 'Admin User',
             'email' => 'admin@'.preg_replace('/https?:\/\//', '', config('app.url')),
             'email_verified_at' => now(),
             'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
```

Comment:
```suggestion
            'password' => Hash::make('password'');
```

(Needs `use Illuminate\Support\Facades\Hash;`)

Reply:
Fixed